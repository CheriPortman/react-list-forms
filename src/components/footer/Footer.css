.Footer {
  text-align: center;
  padding: 1rem;
  background-color: rgb(196, 116, 26);
  border: 3px solid black;
}
