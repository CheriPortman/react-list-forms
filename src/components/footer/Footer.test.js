import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer';

describe('Footer Component', () => {
  it('renders the footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
