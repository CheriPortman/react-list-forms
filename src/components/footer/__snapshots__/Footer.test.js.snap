// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component renders the footer 1`] = `
<footer
  className="Footer"
>
  <p>
    by Cheri
  </p>
</footer>
`;
